pkg/asset/cluster/tfvars.go

```diff
@@ -148,6 +154,19 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 	}
 
 	masterCount := len(mastersAsset.MachineFiles)
+	mastersSchedulable := false
+	for _, f := range manifestsAsset.Files() {
+		if filepath.Base(f.Filename) == "cluster-scheduler-02-config.yml" {
```

Comment:
Should we make `schedulerCfgFilename` in pkg/asset/manifests/scheduler.go public so that we are not duplicating the filename here?

Reply:
I was wondering about it too, and was bother by the fact that schedulerCfgFilename is a var and not a const. That said, it's indeed cleaner to re-use the same var.

Reply:
Unfortunately, it cannot be a const since it is the result of a function call. So, to make it a const, you would have to not use `filepath.Join`